litellm/proxy/guardrails/guardrail_hooks/bedrock_guardrails.py

```diff
@@ -50,6 +51,51 @@
 GUARDRAIL_NAME = "bedrock"
 
 
+def _redact_pii_matches(response_json: dict) -> dict:
+    try:
+        # Create a deep copy to avoid modifying the original response
+        redacted_response = copy.deepcopy(response_json)
+
+        # Get assessments from the response
+        assessments = redacted_response.get("assessments", [])
+        if not assessments:
+            return redacted_response
+
+        for assessment in assessments:
+            # Redact PII entities in sensitive information policy
+            sensitive_info_policy = assessment.get("sensitiveInformationPolicy")
```

Comment:
nit, add a default val like done for assessments and piiEntities

Reply:
Makes sense